source/geoUtils.ts

```diff
@@ -1,80 +1,80 @@
-import * as https from "https";
```

Comment:
Reformatting and type improvements. This will be deprecated in the next version.